products/amazon-glue.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Amazon Glue
+category: service
+tags: amazon glue
+iconSlug: amazonaws
+permalink: /amazon-glue
+releasePolicyLink:
+  https://docs.aws.amazon.com/glue/latest/dg/glue-version-support-policy.html
+releaseDateColumn: true
+releaseColumn: false
+
+auto:
+-   custom: true
+
```

Comment:
```suggestion
```

A corresponding script must be added in https://github.com/endoflife-date/release-data/ for this to work. Let's remove it if you do not plan to develop that script now.